src/main/java/com/strandls/esmodule/services/ElasticSearchService.java

```diff
@@ -28,6 +29,8 @@
  * @author mukund
  */
 public interface ElasticSearchService {
+	
+	List<UploadersInfo> uploaderInfo(String index,String userIds);
```

Comment:
define access modifier public in the deifination

Reply:
added public